consensus-engine/src/overlay/branch_overlay.rs

```diff
@@ -1,4 +1,8 @@
+use fraction::Fraction;
```

Comment:
what are the changes in this file for?

Reply:
Tree overlay was updated with fractions while developing the integration tests. Branch is supposed to use same threshold logic, now both overlays use the same function for that. I can remove this for now.

Reply:
Yes, please do that in a separate PR